crates/nu-command/tests/commands/into_int.rs

```diff
@@ -38,6 +38,34 @@ fn into_int_binary_signed() {
     assert!(actual.out.contains("-16"));
 }
 
+#[test]
+fn into_int_binary_back_and_forth() {
+    let actual = nu!("echo 0x[f0] | into int | into binary | to nuon");
+
+    assert!(actual.out.contains("0x[F000000000000000]"));
+}
+
+#[test]
+fn into_int_binary_signed_back_and_forth() {
+    let actual = nu!("echo 0x[f0] | into int --signed | into binary | to nuon");
+
+    assert!(actual.out.contains("0x[F0FFFFFFFFFFFFFF]"));
+}
+
+#[test]
+fn into_int_binary_empty() {
+    let actual = nu!("echo 0x[] | into int");
+
+    assert!(actual.out.contains('0'))
+}
+
+#[test]
+fn into_int_binary_signed_empty() {
+    let actual = nu!("echo 0x[] | into int --signed");
+
+    assert!(actual.out.contains('0'))
+}
```

Comment:
Should this operation always succeed?

Reply:
the current behavior for `0x[] | into int` is to succeed and return 0, so I kept that behavior for unsigned and signed conversions and also added tests for it which were previously absent (re: https://github.com/nushell/nushell/pull/11902#discussion_r1495032027)